modules/manage/pages/iceberg/iceberg-topics-aws-glue.adoc

```diff
@@ -0,0 +1,191 @@
+
+== Prerequisites
+
+* Redpanda version 25.1.7 or later.
+ifndef::env-cloud[]
+* xref:manage:tiered-storage.adoc#configure-object-storage[Object storage configured] for your cluster and xref:manage:tiered-storage.adoc#enable-tiered-storage[Tiered Storage enabled] for the topics for which you want to generate Iceberg tables.
++
+You also use the S3 bucket URI to set the base location for Glue Data Catalog.
+endif::[]
+
+== Limitations
+
+=== Nested partition spec support
+
+Glue does not support partitioning on nested fields. If Redpanda detects that
+the default partitioning `(hour(redpanda.timestamp))` is in use, it will instead apply an empty partition spec `()`, which means the table will not be partitioned.
+
+If you want to use partitioning, you must specify a custom partition specification using your own partition columns (columns that are not nested).
+
+== Authorize access to AWS Glue
+
+You need to allow Redpanda access to Glue services in your AWS account. You can use the same access credentials that you configured for S3 (IAM role, access keys, and KMS key), as long as you have also added read and write access to Glue Data Catalog.
+
+For example, you could create an IAM policy that manages access to Glue, and attach it to the IAM role that Redpanda also uses to access S3. It is recommended to add all Glue actions in the policy (`"glue:*"`), on the following resources:
+
+- Root catalog (`catalog`)
+- All databases (`database/*`)
+- All tables (`table/*/*`)
+
+Your policy should look similar to the following:
+
+[,json]
+----
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "glue:*"
+      ],
+      "Resource": [
+        "arn:aws:glue:<aws-region>:<aws-account-id>:catalog",
+        "arn:aws:glue:<aws-region>:<aws-account-id>:database/*",
+        "arn:aws:glue:<aws-region>:<aws-account-id>:table/*/*"
+        ]
+    }
+  ]
+}
+----
+
+== Configure authentication and credentials
+
+You can configure credentials for the Glue Data Catalog integration in two ways:
+
+* Allow Redpanda to use the same `cloud_storage_*` credential properties configured for S3. If you do not configure the overrides listed below, Redpanda uses the same credentials for both S3 and Glue. This is the recommended approach.
+* If you want to configure authentication to AWS Glue separately from authentication to S3, there are equivalent credential configuration properties named `iceberg_rest_catalog_*` that override the object storage credentials. These properties only apply to REST catalog authentication, and never to S3 authentication.
+** `iceberg_rest_catalog_aws_access_key` overrides `cloud_storage_access_key`
+** `iceberg_rest_catalog_aws_secret_key` overrides `cloud_storage_secret_key`
+** `iceberg_rest_catalog_aws_region` overrides `cloud_storage_region`
```

Comment:
@Lazin here we have a separate region configuration than the main one used by TS.   So it should be easy enough to do this for RRR topics as well I guess.